### PR TITLE
Marcos.reginato/character fragment dialog

### PR DESCRIPTION
**Pull request by regmarcos**
Hi guys! This PR is about the Character DialogFragment. Now, if you click on an item from the recyclerview in main activity it will show you a DialogFragment with some info about the character. Also, you will see a recyclerview inside the fragment with a list of comics where this character appears.

Here is the evidence of how this fragment works:
https://gfycat.com/idioticsecretlabradorretriever

TODO: show the fragment only when each one of the views are loaded

### CODE REVIEW

File: app/src/main/res/layout/character_fragment_dialog.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.globant.fragments.CharacterFragmentDialog">
+
+    <!-- TODO: Update blank fragment layout -->
```
Comment (SartiMau) [inline app/src/main/res/layout/character_fragment_dialog.xml]: this TODO is still necessary?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -9,4 +9,9 @@
     <string name="get_characters_error">Something went wrong, please try again</string>
     <string name="press_to_show_characters">Press to show Characters</string>
     <string name="name">name</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
```
Comment (SartiMau) [inline app/src/main/res/values/strings.xml]: same

**Review by SartiMau — APPROVED** [review]

File: app/src/main/res/layout/character_fragment_dialog.xml (hunk 1 above)
Comment (regmarcos) [reply app/src/main/res/layout/character_fragment_dialog.xml]: no, its not necessary jajaja i removed it yet

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (regmarcos) [reply app/src/main/res/values/strings.xml]: no, its not necessary jajaja i removed it yet

**Review by SartiMau — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by oadussang — APPROVED** [review]

**Pull request merged**